server/main_test.go

```diff
@@ -0,0 +1,6 @@
+package main
+
+// Unit testi dlya pidorasov
```

Comment:
Please, explain this.

Reply:
We wanted to push the new files as soon as possible to make them accessible to everyone. Since the unit tests were not ready at that moment, we didn't thoroughly check their contents. In the latest version, unnecessary comments have been removed. Moving forward, we will ensure that comprehensive testing is conducted and check every file for unnecessary comments prior to releasing any future updates.